src/engine.py

```
@@ -11606,6 +11610,67 @@ async def _start_health_server(self):
+        if obj is None or isinstance(obj, (str, int, float, bool)):
+            return obj
+        if isinstance(obj, dict):
+            out: Dict[str, Any] = {}
+            for idx, (k, v) in enumerate(list(obj.items())[:max_items]):
+                out[str(k)] = Engine._safe_jsonable(v, max_depth=max_depth, max_items=max_items, depth=depth + 1)
+            return out
+        if isinstance(obj, (list, tuple)):
+            return [Engine._safe_jsonable(v, max_depth=max_depth, max_items=max_items, depth=depth + 1) for v in list(obj)[:max_items]]
+        return str(obj)
+
+    async def _tools_definitions_handler(self, request):
```

Ruff (0.15.0)

[warning] Unused method argument: `request`

(ARG002)